frontend/packages/local-storage-operator-plugin/src/components/local-volume-discovery/body.tsx

```diff
@@ -17,6 +20,11 @@ export const LocalVolumeDiscoveryBody: React.FC<LocalVolumeDiscoveryBodyProps> =
   taintsFilter,
 }) => {
   const { t } = useTranslation();
+  const [lvdrList] = useK8sWatchResource<LocalVolumeDiscoveryResultKind[]>(lvdrResource);
+  const preSelectedNodes = [
+    ...selectNodes.map(getName),
+    ...lvdrList.map((node) => node.spec.nodeName),
+  ];
```

Comment:
Let's move this out of generic component.
Since it will disrupt the flow in wizard. We dont want to show pre running discoveries all the time on wizards page.
```suggestion
  const preSelectedNodes = [
    ...selectNodes.map(getName),
    ...props.preSelectedNodes
  ];
```
And pass `preSelectNodes` of type string[] as a props.
I see having it as `selectNodes` is complicated.

Reply:
@afreen23 In the current implementation since we've moved the watching logic out of the component the discovery is not updated except when the parent component renders. Is that the behavior we're aiming for?